### ISSUE TITLE
how to do when ID different

### ISSUE THREAD

**Issue by zy19900820** [filecoin-project/lotus#2941]
Please provide all the information requested here to help us troubleshoot "commit failed" issues.
If the information requested is missing, we will probably have to just ask you to provide it anyway,
before we can help debug.

**Describe the problem**
what should I do when it happened? This happens all the time. 
Should I return a error in this function ?
if *si.CommR != pci.Info.SealedCID {
		log.Warn("on-chain sealed CID doesn't match!")
	}

It might at after precommitwait and the task restart from P1 somehow(miner restart ,worker down ,program update ...)

**Sectors list**

The output of `./lotus-storage-miner sectors list`.

**Sectors status**

The output of `./lotus-storage-miner sectors status --log <sectorId>` for the failed sector(s).

**Lotus storage miner logs**

Please go through the logs of your storage miner, and include screenshots of any error-like messages you find.

**Version**

lotus version 0.4.3+2k+git.fe7f5e7c.dirty

**Comment by arajasek** [comment]
Digesting the issue a bit, it seems that they see the `on-chain sealed CID doesn't match` warning from [here](https://github.com/filecoin-project/lotus/blob/next/extern/storage-fsm/checks.go#L145) a lot. How much of a problem is this?

**Comment by Reiers** [comment]
Hi @zy19900820 

Thanks for the report. The issue is outdated and Im unable to repro here. 

If this is a issue in the current release - please open a new ticket and provide new logs. 

Thank you !
